src/nulibtable_reader.F90

```diff
@@ -1,17 +1,20 @@
 !-*-f90-*-
-subroutine nulibtable_reader(filename,include_Ielectron,include_epannihil_kernels)
+subroutine nulibtable_reader(filename,include_Ielectron,include_epannihil_kernels,include_scattering_delta)
   
   use nulibtable
   use hdf5
+  use h5lt
```

Comment:
Is h5lt required? If so, not a big deal, it seems like it comes with a standard H5 install.

Reply:
Done.